dashboards-observability/public/components/common/search/autocomplete.tsx

```diff
@@ -399,6 +413,8 @@ export function Autocomplete({
                     },
                     dslService
                   );
+                  $("#autocomplete-textarea").blur();
+                  $("#autocomplete-textarea").focus();
```

Comment:
i put these as workarounds, might be able to get it working by `this.getItems()` or `this.getSources()`, not sure. did you try other ways or this is the only way to trigger suggestion?

Reply:
Yea for some reason nothing can get called because they are all within each other.